docs/release/support_matrix_v2_0.md

```diff
@@ -1,73 +1,273 @@
-# DAOS Version 2.0 Support Matrix
+# DAOS Version 2.0 Support
 
-## Hardware platform support
 
-DAOS is primarily validated and supported on Intel x86\_64 architecture.
+## Community Support and Commercial Support
+
+Community support for DAOS is available through the
+[DAOS mailing list](https://daos.groups.io/) and the
+[DAOS Slack channel](https://daos-stack.slack.com/).
+The [DAOS community JIRA tickets](https://daosio.atlassian.net/jira)
+can be searched for known issues and possible solutions.
+Community support is provided on a best effort basis
+without any guaranteed SLAs.
+
+Starting with DAOS Version 2.0, the Intel DAOS engineering team
+can also be contracted to provide Commercial Level-3 Support for DAOS.
+Under such a support agreement, Intel partners that offer DAOS
+Commercial Support to their end customers will provide the DAOS
+Level-1 and Level-2 support. They can then escalate Level-2 support
+tickets to the Intel Level-3 support team
+through a dedicated JIRA path with well-defined SLAs.
+Please refer to the
+[intel.com landing page for DAOS](https://www.intel.com/content/www/us/en/high-performance-computing/daos.html)
+for information on the DAOS partner ecosystem.
+
+This document describes the supported environments for Intel Level-3 support.
+Partner support offerings may impose further constraints, for example if they
+include DAOS support as part of a more general cluster support offering
+with its own release cycle.
+
+Some members of the DAOS community have reported successful compilation
+and basic testing of DAOS in other environments (for example on ARM64
+platforms, or on other Linux distributions). Those activities are highly
+appreciated community contributions. However such environments are
+not currently supported in a production environment.
+
+
+## Hardware platforms supported for DAOS Servers
+
+DAOS Version 2.0 supports the x86\_64 architecture.
 
 DAOS servers require Storage Class Memory (SCM) that is supported by
 [PMDK](https://github.com/pmem/pmdk).
 (SCM can be emulated by DRAM for development and testing purposes,
 but this is not supported in a production environment.)
-DAOS has been validated with
+DAOS Version 2.0 has been validated with
 [Intel Optane Persistent Memory 100 Series](https://ark.intel.com/content/www/us/en/ark/products/series/190349/intel-optane-persistent-memory-100-series.html)
-on 2nd gen Intel Xeon Scalable processors, and
+on 2nd gen Intel Xeon Scalable processors, and with
 [Intel Optane Persistent Memory 200 Series](https://ark.intel.com/content/www/us/en/ark/products/series/203877/intel-optane-persistent-memory-200-series.html)
 on 3rd gen Intel Xeon Scalable processors.
+It is strongly recommended that all memory channels of a DAOS server
+are populated with one DRAM module and one Optane PMem module.
```

Comment:
Should we say *why*? Something like "It is strongly recommended that all memory channels of a DAOS server are populated with one DRAM module and one Optane Pmem module for maximum performance." - just something to convey that the consequence is loss of performance if not fully populated.